Modules/CitiesViewController/Cells/CityTableCell.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  CityTableCell.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 04/04/2024.
+//
+
+import UIKit
+
+final class CityTableCell: UITableViewCell {
+    
+    lazy var labelsStack = { 
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.spacing = 10
+        return stack
+    }()
+    
+    var pageTitleLbl = ReusableBoldLabel()
+    var pageSecondaryLbl = ReusableBoldLabel() {
+        didSet {
+            pageSecondaryLbl.isHidden = true
+        }
+    }
+
+    static let identifier = "CityTableCell"
+    
+    // MARK: initialiser
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        setViewSpecs()
+        setConstaints()
+        
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: Set view props
+    private func setViewSpecs() {
+        pageTitleLbl.setAlignment(.natural)
+        pageSecondaryLbl.setAlignment(.natural)
+
+        accessoryType = .detailDisclosureButton
+//        accessoryView = UIImageView(
+//            image: UIImage(systemName: "chevron.right")
+//        )
+        tintColor = Colors.mainBtnBackgroundColor
+        pageSecondaryLbl.textColor = Colors.mainBtnBackgroundColor
+        backgroundColor = .clear
+        selectionStyle = .none
+    }
+    
+    func configWith(
+        _ model: CityTableCell.CellModel,
+        _ primaryFont: UIFont,
+        _ secondaryFont: UIFont
```

Comment:
* I expect fonts are added once, so you don't have to configure them every time. 
* If reused, please add them in the view model. Ideally, a method should have no inputs, one if needed.